### DISCUSSION TITLE
Performance differences from 0.13.2 to 0.14-beta3

### DISCUSSION THREAD

**Discussion by PovilasID** [blakeblackshear/frigate#12089] [General]
Hey,

I transition from 0.13.2 to 0.14 beta 3. I am loving the new UI very well though out design to quickly check if anything new happened.
However, I noticed that the new version requires a bit more resources namely RAM.

HW specs:
CPU i3-6100
RAM 8 GB 
SSD 128 GB and 1 TB HDD
4 cameras* (one needs replacement because it keeps cutting off and one is indoors so does not see a lot of motion like trees moving )
I am using OpenVINO accleraotr.
```
detectors:
  ov:
    type: openvino
    device: GPU
    model:
      path: /openvino-model/ssdlite_mobilenet_v2.xml
```
I actually got coral TPU but need to find longer screws to put the half lengh bord in...

Frigate runs in docker container, I run frigate Prometheus exporter on remote server that uses frigate's own API and permits viewing on Grafan. 

RAM use increased for 'Other processes'  
![image](https://github.com/blakeblackshear/frigate/assets/396243/20991ac5-4471-4d1e-b920-c07440dc9356)
You can see it dipping after a first and second restart but I think that slowly creeps up as especially as I explore the new UI. I am fine with frigate using more RAM however it would be cool if frigate was more efficient maybe a "headless" version for remote deploys.

CPU use mean has also gone up:
![image](https://github.com/blakeblackshear/frigate/assets/396243/c89a6559-1124-41f3-bba8-7d7b0e9d5242)
You can see that the night time is ofc less busy so less processing needs to be done but the floor is higher, however, this can probably be explained by increase in frame detections for that period:
![image](https://github.com/blakeblackshear/frigate/assets/396243/93993093-3255-44ba-a987-26bc443a6cba)
There is also an increase in processes count from ~52->60

I am feeding frigte full 1080p streams against the recommendation because my gate is far away from my door and in lower res stream just does not provide enough pixels to identify the mail man passing by or a guy jamming his screwdriver into my gate lock.

**Comment by hawkeye217** [comment]
The increase in process count is expected due to changes in 0.14 with the new review format.

The memory increase is not something we've seen, nor has it been reported by most users on 0.14 beta. The exception is for users running frigate in Proxmox as there has been a kernel issue with Proxmox that has manifested itself as a memory leak.

Since we can't reproduce your issue, the only way we'll be able to help is if you can try to narrow down the issue on your end. Try removing cameras (especially your non-functioning ones) from your config, changing detectors, minimizing your config to one camera and letting frigate run, etc.  Graphs are nice but the more info you can give us, the better the chance is we'll be able to help find your particular issue.

**Reply by NickM-27** [reply]
it's especially helpful to be able to see the system metrics pages (and the system page from the old UI) as this gives specific information about what in frigate is using what memory and potentially allows for comparing

**Reply by kirsch33** [reply]
@PovilasID what proxmox kernel version do you have? I've been seeing a few proxmox users reporting this issue lately but havent been experiencing it myself so im wonderng if its related to a newer/recently released kernel version that I'm not running. That would help to narrow it down.

**Reply by PovilasID** [reply]
@kirsch33 I am not running on proxmox machine. Host is running on OpenWRT x86 system. 
Last RAM and cpu use climbs over time to 100. I am a little busy now to give full feedback but here is the copy of what I see when I press Hardware info in general stats page:
```
{"return_code":0,"stderr":"","stdout":"vainfo:VA-APIversion:1.17(libva2.10.0)nvainfo:Driverversion:InteliHDdriverforIntel(R)GenGraphics-23.1.1()nvainfo:SupportedprofileandentrypointsnVAProfileNone:tVAEntrypointVideoProcnVAProfileNone:tVAEntrypointStatsnVAProfileMPEG2Simple:tVAEntrypointVLDnVAProfileMPEG2Simple:tVAEntrypointEncSlicenVAProfileMPEG2Main:tVAEntrypointVLDnVAProfileMPEG2Main:tVAEntrypointEncSlicenVAProfileH264Main:tVAEntrypointVLDnVAProfileH264Main:tVAEntrypointEncSlicenVAProfileH264Main:tVAEntrypointFEInVAProfileH264Main:tVAEntrypointEncSliceLPnVAProfileH264High:tVAEntrypointVLDnVAProfileH264High:tVAEntrypointEncSlicenVAProfileH264High:tVAEntrypointFEInVAProfileH264High:tVAEntrypointEncSliceLPnVAProfileVC1Simple:tVAEntrypointVLDnVAProfileVC1Main:tVAEntrypointVLDnVAProfileVC1Advanced:tVAEntrypointVLDnVAProfileJPEGBaseline:tVAEntrypointVLDnVAProfileJPEGBaseline:tVAEntrypointEncPicturenVAProfileH264ConstrainedBaseline:tVAEntrypointVLDnVAProfileH264ConstrainedBaseline:tVAEntrypointEncSlicenVAProfileH264ConstrainedBaseline:tVAEntrypointFEInVAProfileH264ConstrainedBaseline:tVAEntrypointEncSliceLPnVAProfileVP8Version0_3:tVAEntrypointVLDnVAProfileVP8Version0_3:tVAEntrypointEncSlicenVAProfileHEVCMain:tVAEntrypointVLDnVAProfileHEVCMain:tVAEntrypointEncSlicenVAProfileHEVCMain:tVAEntrypointFEI"}
```
